src/components/concentration-display/ConcentrationSlider.tsx

```diff
@@ -87,7 +90,7 @@ const ConcentrationSlider: React.FC<SliderProps> = ({
             name={name}
             min={min}
             max={max}
-            step={2}
+            step={stepSize.current}
```

Comment:
the second module has a much bigger range, so we're showing bigger jumps between markers